internal/distro/fedora/package_sets.go

```diff
@@ -532,7 +532,7 @@ func containerPackageSet(t *imageType) rpmmd.PackageSet {
 	return ps
 }
 
-func minimalrpmPackageSet(t *imageType) rpmmd.PackageSet {
+func minimalRawImgPackageSet(t *imageType) rpmmd.PackageSet {
```

Comment:
I just noticed that this package set is used also by the image installer image type, so keeping the old name should be better. This seems to be the case only for Fedora. Sorry for not noticing during my initial review :innocent:

Reply:
ahh!!  is it okay if I change the name in all the places its been used as a part of this PR ?

Reply:
sure. sorry for the noise...